troubleshooting.adoc

```diff
@@ -82,6 +82,8 @@ Check the status.decision list contains an entry for the managed cluster.  If no
 +
 image::./images/authentication_operator_error.png["authentication operator error message indicates certificate error"]
 +
+The above message means the certificate on the hub is not well set, please refer to xref:./requirements.adoc##openid-server-certificate[Securing OAuth requests to the OpenID Connect identity provider]
```

Comment:
I wonder if we should spell this out a little more, as it relates specifically to the error message shown in the picture, and does not cover all possible issues you would find here, e.g.:
The error message seen above, "x509: certificate signed by unknown authority", indicates the certificate on the hub is not valid. Refer to .. for details.

It looks like the numbering in this section is not correct anymore... The following item used to be 4 and is now showing as 1. Please take a look.